src/engine/strat_engine/serde_structs.rs

```diff
@@ -37,6 +37,7 @@ pub struct PoolSave {
     pub backstore: BackstoreSave,
     pub flex_devs: FlexDevsSave,
     pub thinpool_dev: ThinPoolDevSave,
+    // TODO: This data type should no longer be optional in Stratis 4.0
```

Comment:
Was removing this comment intentional? I think it should still be there.

Reply:
I think you are reading it backwards. I added this comment, because I believe we intended to make the started field mandatory in Stratis 4.0.

Reply:
Whoops! I'll approve this.